src/style/homepage/styled.js

```diff
@@ -0,0 +1,162 @@
+import styled from "styled-components";
+import imgBackgroundMobile from "../../assets/home/background-home-mobile.jpg";
+import imgBackgroundTablet from "../../assets/home/background-home-tablet.jpg";
+import imgBackgroundDesktop from "../../assets/home/background-home-desktop.jpg";
+
+export const Container = styled.main`
+  background-image: url(${imgBackgroundMobile});
+  background-position: bottom;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (min-width: 768px) {
+    background-image: url(${imgBackgroundTablet});
+  }
+  @media (min-width: 1028px) {
```

Comment:
1028px is usually used as a tablet size, desktop breakpoint could start from 1280px